### PR TITLE
[5.4] Add more information to migrations

### PR DESCRIPTION
**Pull request by gsaqui**
When running a number of migrations it will give information on which migration is currently running and when the migration is finished it will say how many seconds it took to run. https://github.com/laravel/internals/issues/436

```
Running: 2014_10_12_000000_create_users_table...Done! (2s)
Running: 2014_10_12_100000_create_password_resets_table...Done! (0s)
```

### CODE REVIEW

**Comment by gsaqui** [comment]
Do I need to fix the styles?  I thought StyleCI dealt with it after the merge was committed.

**Review by pjona — APPROVED** [review]

**Comment by sailingdeveloper** [comment]
@gsaqui Yes, you do :)

**Comment by stefanoruth** [comment]
Dont really see the need with migrations but with seeders it could be nice :)

File: src/Illuminate/Database/Console/Migrations/MigrateCommand.php [hunk 1]
```diff
@@ -64,17 +64,12 @@ public function fire()
         // Next, we will check to see if a path option has been defined. If it has
         // we will use the path relative to the root of this installation folder
         // so that migrations may be run for any path within the applications.
+
```
Comment (fernandobandeira) [inline src/Illuminate/Database/Console/Migrations/MigrateCommand.php]: Blank like

**Comment by taylorotwell** [comment]
No plans to add this for now.

**Comment by gsaqui** [comment]
@taylorotwell not really sure where the break down of communication happened.  I raised this issue https://github.com/laravel/internals/issues/436 and then asked if this was something that would be accepted on https://larachat.slack.com/messages/internals and assumed that this was something that the community wanted. Where did I go wrong so next time this doesn't happen?


@stefanoruth the reason I wrote this was because I have long running migrations and I'm never sure which one is running at the time and wanted the framework to give me some indication. The current situation is you don't know what is happening until the end when you get an output of the migrations that ran.

**Pull request closed without merging**